R/calcOutput.R

```diff
@@ -217,7 +226,7 @@ calcOutput <- function(type, aggregate = TRUE, file = NULL, years = NULL, round
   on.exit(setwd(cwd)) # nolint
   if (is.null(getOption("gdt_nestinglevel"))) vcat(-2, "")
   startinfo <- toolstartmessage(argumentValues, "+")
-  on.exit(toolendmessage(startinfo, "-"), add = TRUE)
+  on.exit(toolendmessage(startinfo, "-"), add = TRUE) # nolint
```

Comment:
```suggestion
  defer(toolendmessage(startinfo, "-"))
```
Oben dann noch `#' @importFrom withr defer`

Reply:
Bin mir nicht sicher ob das hier kaputt gehen würde, aber da du ja .local verwenden willst (was auf withr basiert) würde ich hier kein on.exit haben wollen